Packs/EWS/Integrations/EWSv2/EWSv2.py

```diff
@@ -898,14 +902,38 @@ def get_last_run():
     return last_run
 
 
+def parse_fetch_time_to_minutes():
```

Comment:
Are you sure there isn't existing functionality for this in CommonServerPython?

Reply:
Yes, we should use dateparser.parse instead.